README.rst

```diff
@@ -44,6 +44,16 @@
 - Rich README template containing badges, installation, support, and contribution guide.
 - Automatic spelling check, linting for .json, .yml, and .md files.
 
+Demo
+----
+
+.. image:: doc/source/gif/demo.gif
```

Comment:
love the gif, but I feel that it is going a bit too quickly.  Is there any way to slow down the frame-rate some?